test/gurka/test_avoids.cc

```diff
@@ -187,7 +187,7 @@ TEST_P(AvoidTest, TestAvoid2Polygons) {
   };
 }
 
-TEST_P(AvoidTest, TestAvoidShortcuts) {
+TEST_F(AvoidTest, TestAvoidShortcutsTruck) {
```

Comment:
@nilsnolde Tagged you on the PR as you showed up in `git blame`. I'm tweaking the test to `TEST_F` so it doesn't run for each of the parameters at the end of the file. This test is hard-coded to use truck costing and does not refer to the `GetParameter()` function.

Reply:
yep thanks, looks like c/p fail